README.md

```diff
@@ -95,9 +95,9 @@ Sample skill that shows how to request and access the configured address in the
 Project that demonstrates how to use the audio player for skills.
 
 ## Other Language Alexa Skills Kit SDKs
-<a href="https://github.com/amzn/alexa-skills-kit-java"><img src="https://github.com/konpa/devicon/raw/master/icons/java/java-original.svg?sanitize=true" width="25px" /> Alexa Skills Kit SDK for Java</a>
+<a href="https://github.com/alexa/alexa-skills-kit-java"><img src="https://github.com/konpa/devicon/raw/master/icons/java/java-original.svg?sanitize=true" width="25px" /> Alexa Skills Kit SDK for Java</a>
```

Comment:
The correct link should be https://github.com/alexa/alexa-skills-kit-sdk-for-java